docs/modules/blocks/enderchests.mdx

```diff
@@ -9,12 +9,11 @@ The `<enderchest>` tag can contain any number of `<dropoff>` tags. Each `<dropof
 determine where and when items should be dropped from the Ender chest.
 
 Dropoffs handle taking a player's Ender chest inventory and dropping it on the ground at a defined region or it can completely erase it.
-This can be triggered with a filter or once the player switches team or leaves the server.
+This is triggered once the player switches team or leaves the server.
```

Comment:
Technically if its they leave the match, or "stop participating" in the match (eg: /obs), but its quite a small detail i think most people wont notice the difference

Reply:
Reworded to `This is triggered once the player switches teams or stops participating in the match.`